### PR TITLE
Adding Nightly and Android shortcuts

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Review by dru89 — COMMENTED** [review]
Given that these are urls related directly to Firefox (and the protocol is specifically related to its parent, Mozilla), wouldn't it make more sense for the keys to be `firefox/nightly` and `firefox/android`?

**Comment by Natim** [comment]
Sure I can change that 👍

**Comment by dru89** [comment]
I don't work on this project, was just a thought I had reading through this. I'd definitely wait for a maintainer if you were worried, but I think the hierarchies make sense in this case. :smile:

**Review by mykmelez — APPROVED** [review]
Thanks for the submission, and sorry about the delay reviewing! (My GitHub account wasn't "following" the repo, even though I created it, so I didn't get notified about the request.)

These additions look great, and I'll merge them with these changes:

* point firefox/nightly directly to the page that nightly.mozilla.org now redirects to
* point firefox/android to the page for the release version of Firefox for Android
* remove the language from the firefox/android URL (so mozilla.org redirects to the user's preferred language)
* ensure both links are https

**Comment by Natim** [comment]
👍 thanks

File: docs/index.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
 const urls = {
   '':                 'https://mozilla.org',
   'firefox':          'https://firefox.com/',
+  'firefox/android':  'https://www.mozilla.org/firefox/android/',
+  'firefox/nightly':  'https://www.mozilla.org/firefox/channel/desktop/#nightly',
```
Comment (Natim) [inline docs/index.html:8]: If you use nightly.mozilla.org from android it will redirect you to the nightly android page.

**Comment by mykmelez** [comment]
> If you use nightly.mozilla.org from android it will redirect you to the nightly android page.

Ah, good point, I didn't realize that! Ok, I've changed firefox/nightly back to https://nightly.mozilla.org/ over in #9.

**Pull request merged**